workspace/notebook/py_create_lake_databases.json

```diff
@@ -86,36 +86,37 @@
 					}
 				},
 				"source": [
-					"#raw_database_name = 'odw_raw_db'\r\n",
-					"#raw_database_location = 'abfss://odw-raw@' + storage_account\r\n",
-					"#workspace_database_name = 'odw_workspace_db'\r\n",
-					"#workspace_database_location = 'abfss://odw-workspace@' + storage_account + '/'\r\n",
-					"standardised_database_name = 'odw_standardised_db'\r\n",
-					"standardised_database_location = 'abfss://odw-standardised@' + storage_account + '/'\r\n",
-					"harmonised_database_name = 'odw_harmonised_db'\r\n",
-					"harmonised_database_location = 'abfss://odw-harmonised@' + storage_account + '/'\r\n",
-					"curated_database_name = 'odw_curated_db'\r\n",
-					"curated_database_location = 'abfss://odw-curated@' + storage_account + '/'\r\n",
-					"config_database_name = 'odw_config_db'\r\n",
-					"config_database_location = 'abfss://odw-config@' + storage_account + '/'\r\n",
-					"\r\n",
-					"curated_migration_database_name = 'odw_curated_migration_db'\r\n",
-					"curated_migration_database_location = 'abfss://odw-curated-migration@' + storage_account + '/'"
+					"logging_database_location = f\"abfss://logging@{storage_account}\"\r\n",
+					"standardised_database_location = f\"abfss://odw-standardised@{storage_account}\"\r\n",
+					"harmonised_database_location = f\"abfss://odw-harmonised@{storage_account}\"\r\n",
+					"curated_database_location = f\"abfss://odw-curated@{storage_account}\"\r\n",
+					"curated_migration_database_location = f\"abfss://odw-curated-migration@{storage_account}\"\r\n",
+					"config_database_location = f\"abfss://odw-config-db@{storage_account}\""
```

Comment:
think this should be `odw-config` instead of  `odw-config-db`

Reply:
No, we already have a container called odw-config for the orchestration file and config for tables. Therefore I gave it a new name.